apps/omg/test/support/test_helper.ex

```diff
@@ -154,10 +154,10 @@ defmodule OMG.TestHelper do
   @doc """
   Always creates file in the priv/ folder of the application.
   """
-  @spec write_fee_file(%{Crypto.address_t() => map()} | binary(), binary() | nil) :: {:ok, binary, binary}
-  def write_fee_file(fee_map, file_name \\ nil)
+  @spec write_fee_file(%{Crypto.address_t() => map()} | binary()) :: {:ok, binary, binary}
+  def write_fee_file(fee_map, file_path \\ nil)
```

Comment:
what purpose does this serve though?

Reply:
The fee specs file is supposed to be updatable but it's residing deep in `_build/.../<app-version>/.../omg_child_chain/priv` folder making it hard and error prone to update.

So going for a path instead so infra could keep the file somewhere else outside the repo directory and we can refer to it via env var.

Reply:
do we stil need the default arument? `file_path \\ nil`?

Reply:
Looks like we still do because https://github.com/omisego/elixir-omg/blob/master/apps/omg_child_chain/test/omg_child_chain/integration/fee_server_test.exs specifically tests the behaviour when the same file gets updated.